src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -96,13 +89,20 @@ const AttritionTableRow = ({
 
   const determineChartIcon = () => {
     if (rowType === 'Covariate' || rowType === 'Outcome') {
-      if (rowObject.variable_type === 'concept') {
+      if (
+        rowObject.variable_type === 'concept'
+        || 'concept_type' in rowObject
```

Comment:
The rowObject was missing the ```variable_type``` key and was causing the application to crash. I updated this to also check for the ```concept_type```to determine if a bar chart should be shown here as a work around.